netlogo-gui/src/main/app/infotab/InfoFormatter.scala

```diff
@@ -44,12 +44,12 @@ object InfoFormatter {
                          MaxParsingTimeMillis)
       .markdownToHtml(str)
 
-  def wrapHtml(body: HTML, fontSize: Int = defaultFontSize): HTML = {
-    "<html><head>"+styleSheet(fontSize)+"</head><body>"+body+"</body></html>"
-  }
+  def wrapHtml(body: HTML, fontSize: Int = defaultFontSize): HTML =
+    "<html><head>"+styleSheet(fontSize)+"</head><body>"+
+      body.replaceAll("<li>", "<li>&bull; ")+
```

Comment:
Did you check whether the generated HTML contained corresponding `</li>` tags? If it does, I would also like those removed so that we have valid HTML both before and after this transformation.